pad_api_data/pad_etl/common/pad_util.py

```diff
@@ -143,467 +143,468 @@ def __init__(self):
 def parse_skill_multiplier(skill, other_fields, length) -> Multiplier:
     multipliers = Multiplier()
 
-    if skill == 3:
-        multipliers.shield = get_last(other_fields)
-
-    # Attack boost only
-    elif skill in [11, 22, 26, 31, 40, 66, 69, 88, 90, 92, 94, 95, 96, 97, 101, 104, 109, 150]:
-        multipliers.atk *= get_last(other_fields)
-
-    # HP boost only
-    elif skill in [23, 30, 48, 107]:
-        multipliers.hp *= get_last(other_fields)
-
-    elif skill in [24, 49, 149]:
-        multipliers.rcv *= get_last(other_fields)
-
-    # RCV and ATK
-    elif skill in [28, 64, 75, 79, 103]:
-        multipliers.atk *= get_last(other_fields)
-        multipliers.rcv *= get_last(other_fields)
+    if length > 0:
```

Comment:
The change in this file is waaaaay too big to review cleanly.

In general, if you're refactoring / cleaning up that should be in it's own CL, separate from changes to fix bugs / add features.